### ISSUE TITLE
Proper repository.

### ISSUE THREAD

**Issue by begin-theadventure** [LinuxBeaver/LinuxBeaver#6]
Would it be possible to create a new proper repository with the source code, documents and tags+releases in it? Something like "GEGLBeaver", and instead use this repository for a profile description, so to mention that GEGL plugins have been moved to a [different repository] (link) etc.

Such a repo could be easier for contributing.

**Comment by begin-theadventure** [comment]
I could make a template, if you'd like.

**Comment by tsal** [comment]
I agree - this current structure just makes it clunky to find the source / appropriate structure to just clone the repo and build it locally, as I think most folks who use github regularly would expect. This feels awkward enough to (almost) not want to use it. I don't typically install binaries unsigned, so building locally is the only option for me.

**Comment by LinuxBeaver** [comment]
I will work on it. Everything is organized here

https://github.com/LinuxBeaver/LinuxBeaver/releases/tag/Gimp_GEGL_Plugin_download_page
